### ISSUE TITLE
Can you provide the command to train models for Pascal-5i, COCO-20i, and fss?

### ISSUE THREAD

**Issue by RenShuhuai-Andy** [isl-org/lang-seg#36]

**Comment by Boyiliee** [comment]
Hi, @RenShuhuai-Andy,

Thanks so much for your interest in LSeg!

We provide the train script for ADE20k dataset, and you could easily revise it for zero-shot experiments. As for FSS-1000, it should be very easy to reproduce the results. As for COCO and PASCAL dataset, due to very few classes, you need early stop (you should be able to get the optimal results using the models from epoch 0-3) and do a hyper-parameter sweep to find the best learning rate, the optimal lr should be smaller than the lr of FSS-1000.

Hope this helps!

Best,
Boyi
